scripts/umbrel-os/status-server/static/index.html

```diff
@@ -67,28 +67,28 @@
       </svg>
 
       <div class="error-text monitor-check">
-        <p><b class="text-dark">Error: External drive not connected</b></p>
-        <p>This can sometimes be caused by using an unofficial power supply.</p>
+        <p><b class="text-dark">Error: External drive disconnected</b></p>
+        <p>The external drive was disconnected while Umbrel was running. This can sometimes happen when using an unofficial Raspberry Pi power supply.</p>
       </div>
 
       <div class="error-text semver-mismatch">
-        <p><b class="text-dark">Error: Can't upgrade from SDcard due to version mismatch</b></p>
-        <p>Can't upgrade from SDcard due to version mismatch.</p>
+        <p><b class="text-dark">Error: Version mismatch</b></p>
+        <p>The version of UmbrelOS on your microSD card is different than the version of Umbrel on your external drive.</p>
```

Comment:
Maybe we can clarify that the reason is that the installed version is too old to be directly upgraded? The error sounds like the issue is just that the versions are different.

```
The version of Umbrel OS on your microSD card is too recent to safely upgrade the version of Umbrel installed on your external drive.
```